chcemvediet/apps/inforequests/tests/__init__.py

```diff
@@ -39,15 +39,16 @@ def mock_render(*args, **kwargs):
             return res
         def mock_render_to_string(*args, **kwargs):
             if pre_mock_render_to_string: # pragma: no cover
-                self.pre_mock_render_to_string(*args, **kwargs)
```

Comment:
Zaujimave ze v case ked to bolo napisane tak testy zbehli. Respektive asi sa nikdy nedefinovala vlastna funckia pre `pre_mock_render_to_string`, t.j. `pre_mock_render_to_string = kwargs.pop(u'pre_mock_render_to_string', None)` vratilo vzdy `None`